src/util/utils.h

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
It may make more sense to name this file `util.h` to be more consistent with the directory name.

Reply:
Actually, the directory should probably be renamed `utils` since the other directories are plural (`modules`); so might refactor that later -- for now Ill leave it until we make the whole code base consistent on that